client/dashboard/app/analytics/index.tsx

```diff
@@ -22,7 +24,19 @@ export function AnalyticsProvider( { children, client }: AnalyticsProviderProps
 }
 
 export function useAnalytics() {
-	return useContext( AnalyticsContext );
+	const context = useContext( AnalyticsContext );
+
+	if (
+		! config< string >( 'env_id' ).includes( 'production' ) &&
+		context === defaultNoopAnalyticsClient
+	) {
+		// eslint-disable-next-line no-console
+		console.error(
+			'useAnalytics() hook expects the <AnalyticsProvider> to be available in the component tree'
```

Comment:
Super minor, but I think the standard pattern for this is "{hook} must be used within a {provider}". Not sure how that started.